app/Http/Controllers/ParticipationController.php

```diff
@@ -72,20 +73,25 @@ public function create($id, Request $request)
         $participation->fill($data);
         $participation->save();
 
-        if ($is_web) {
-            return Redirect::back();
-        } else {
+        if (! $is_web) {
             if ($event->activity->isFull() || ! $event->activity->canSubscribe()) {
                 $message = 'You have been placed on the back-up list for '.$event->title.'.';
             } else {
                 $message = 'You claimed a spot for '.$event->title.'.';
             }
-            abort(200, json_encode((object) [
+
+            return Response::JSON([
```

Comment:
Isn't it just Response::json() or response()->json()? Without using the "use Symfony\Component\HttpFoundation\Response;" but "use Response;" instead?